templates/simple-0/template.es6.js

```diff
@@ -0,0 +1,8 @@
+
+class BespokeTemplate {
+  exec(info) {
+    return `Hello {info.name}! <strong>You have {info.messageCount} messages! {info.colors}`;
```

Comment:
You used `{ }` instead of `${ }` so there was no string interpolation happening. I also dropped the usage of the `class` and just switched to the following:

```js
module.exports = (data) => {
  return `Hello ${data.name}! <strong>You have ${data.messageCount} messages! ${data.colors}`;
};
```

Reply:
Doh! That was copy/paste from the lodash template.

Reply:
Oh also FYI I noticed a slightly higher performance when I ditched the `class` in favor of this:

``` js
function BespokeTemplate() {}
BespokeTemplate.prototype.exec = function exec(data) {
  return `Hello ${data.name}! <strong>You have ${data.messageCount} messages! ${data.colors}`;
}
```

There are some V8 optimizations from the Crankshaft era that under the covers highly optimizes prototype functions.